{"uuid": "c69f2170-1eeb-4d6c-8fc2-b91f6e3d0028", "children": ["5e8ea906-862e-4d70-9a54-3d4b0bbd4b26"], "befores": [{"name": "test_setup", "status": "broken", "statusDetails": {"message": "AttributeError: 'Login' object has no attribute 'loginBeneficiario'\n", "trace": "  File \"C:\\Users\\cmarcano\\OneDrive - Capgemini\\Desktop\\Automation\\SAES\\venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\cmarcano\\OneDrive - Capgemini\\Desktop\\Automation\\SAES\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\cmarcano\\OneDrive - Capgemini\\Desktop\\Automation\\SAES\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"C:\\Users\\cmarcano\\OneDrive - Capgemini\\Desktop\\Automation\\SAES\\tests\\conftest.py\", line 47, in test_setup\n    login_beneficiario.loginBeneficiario(user, password)\n    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n"}, "start": 1699305810972, "stop": 1699305819024}], "start": 1699305810972, "stop": 1699305819598}